build-tools/packages/build-cli/test/commands/generate/buildVersion.test.ts

```diff
@@ -16,19 +17,46 @@ const test_tags = [
 	"client_v1.2.4",
 	"client_v1.2.3",
 	"client_v2.0.0-rc.1.0.0",
+	"client_v2.0.0",
 	"client_v2.0.0-rc.1.0.1",
+	"client_v2.0.1",
 	"client_v2.0.0-rc.2.0.0",
+	"client_v2.1.1",
+	"client_v2.0.0-rc.5.0.0",
 	"client_v2.0.0-rc.3.0.0",
 	"client_v2.0.0-rc.4.0.0",
-	"client_v2.0.0-rc.5.0.0",
-	"client_v2.0.0-rc.6.0.0",
+	"client_v2.1.0",
 	"build-tools_v0.5.2002",
 	"build-tools_v0.4.2001",
-	"build-tools_v0.4.2000",
+	"client_v2.0.0-rc.6.0.0",
 	"build-tools_v0.4.1000",
 	"build-tools_v0.3.2000",
+	"build-tools_v0.4.2000",
 ];
 
+/**
+ * Convenience function to check if a particular line of stdout output equals the expected value.
+ *
+ * @param stdout - the complete stdout string.
+ * @param lineIndex - the index of the line to check.
+ * @param testValue - the value to test against
+ * @returns An assertion that will fail if the line doesn't match the value and pass if it does.
+ */
+function stdoutLineEquals(
+	stdout: string,
+	lineIndex: number,
+	testValue: string,
+): Chai.Assertion {
+	const lines = stdout.split(EOL);
+	if (lineIndex > lines.length) {
+		console.error(lines);
+		throw new Error(
+			`stdout only split into ${lines.length} lines, but lineIndex is ${lineIndex}.`,
+		);
+	}
+	return expect(lines[lineIndex]).equals(testValue);
```

Comment:
My best gut feeling is that this behaves as expected, but I'm left a bit confused that we have to return this. Seems like the return value has no effect, and the failure case will just throw from here...? Otherwise I don't see how the `it()` test would fail (whether using this helper function or directly with an `expect().bla().bla()`.

Reply:
I think you're right that it doesn't need to be returned. Fixed.